src/app.py

```diff
@@ -2,11 +2,35 @@
 from PyQt5.QtWidgets import QApplication
 from engines import ImportDataEngine
 from state import State
+from database import Reader, Writer, DocumentRemover
+from data_import import DatabaseReader
 
 from widgets import MainWindow
 
 
 def main():
+    # uncomment below to show database in action
+    # writer = Writer("db", "collection")
+    # for i in range(11):
+    #     writer.add_document({"key": "value", "another_key": 1})
+    # reader = Reader("db", "collection")
+    # print(reader.get_rows_number()) # number of rows example
+    # print(reader.get_columns_names()) # columns headers
+    # print(reader.execute_query(columns=["key", "another_key"])) # show specified columns only
+    # print(reader.get_nth_chunk(columns=["key", "another_key"])) # nth-chunk with chunk_size=0 and chunk_number=0 returns all records in dataframe
+    # print(reader.execute_query(query={"another_key": 1})) # query example
+    # print(reader.get_nth_chunk()) # all records, equivalent to reader.execute_query, but result is in dataframe
+    # print(reader.get_nth_chunk(use_id=1, columns=["key", "another_key"])) # get id
```

Comment:
Not a comment to this pull request, but we could write a unit test that is similar to this, but asserts equality instead of printing.

Reply:
I'll initialize the directory for testing when this branch has been merged to develop.